src/pages/iou/IOUModal.js

```diff
@@ -456,8 +457,16 @@ class IOUModal extends Component {
                                         >
                                             {this.renderHeader()}
                                             <IOUConfirmPage
-                                                onConfirm={this.createTransaction}
-                                                onSendMoney={this.sendMoney}
+                                                onConfirm={(selectedParticipants) => {
+                                                    if (this.creatingIOUTransaction) { return; }
```

Comment:
```suggestion
                                                    // Prevent creating multiple transactions if the button is pressed repeatedly
                                                    if (this.creatingIOUTransaction) { return; }
```
Let's add a comment here to clarify the change.